includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -31,12 +31,12 @@ public function getHref( $name, $lang ) {
 
 	private $hrefs = [
 		'default' => [
-			'fan-communities' => 'http://fandom.wikia.com/explore',
+			'explore-wikis' => 'http://fandom.wikia.com/explore',
 			'about' => 'http://www.wikia.com/about',
 			'careers' => 'https://careers.wikia.com',
 			'press' => 'http://fandom.wikia.com/press',
 			'contact' => 'http://fandom.wikia.com/contact',
-			'wikia-gives-back' => 'http://www.wikia.com/wikiagivesback',
+			'wikia-org' => 'http://wikia.org',
 			'terms-of-use' => 'http://www.wikia.com/Terms_of_use',
 			'privacy-policy' => 'http://www.wikia.com/Privacy_Policy',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "privacy-policy" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)